### ISSUE TITLE
List of improvements and extra features✨🔥

### ISSUE THREAD

**Issue by euhidaman** [euhidaman/zomato-clone#1]
The following things can be improved 😄 : 
- [x] Header is partially responsive.
- [ ] Header is fully responsive.
- [x] Website body is fully responsive.
- [ ] Making website fully mobile responsive.
- [ ]  Adding more features to the website.
- [x] Adding search functionality.
- [ ] Adding Dark Mode.

Also, please **Star** the repository. So that more people can see it in their feed, and contribute to it. ⭐
And, **before making a pull request**, please make sure to do **fetch upstream**, to remove any merge conflicts.

**Comment by daptheHuman** [comment]
what kind do you like to improve?

**Comment by euhidaman** [comment]
The website is not mobile friendly and responsive right now.
So, those aspects can be improved.

**Comment by NirajGohel** [comment]
@euhidaman this is kinda too big a task. can I do this in smaller parts? like header, footer, etc.  If this is okay with you can you assign this to me?

**Comment by euhidaman** [comment]
**This issue will not be assigned to any individual. Those who want to work on it, can directly fork this repo, make the necessary changes and then create a pull request**

**Note : This is an open issue, It won't be assigned to anyone.**<br>


And, if anyone wants to work on a more beginner friendly repository, then I'm maintaining one more repository for Hacktoberfest. 
Interested developers can contribute on that : [https://github.com/euhidaman/Fake_APIs](https://github.com/euhidaman/Fake_APIs)

**Comment by NirajGohel** [comment]
ok

**Comment by Shivam2001-programmer** [comment]
Please assign to me  this project I want to work it

**Comment by euhidaman** [comment]
**This issue will not be assigned to any individual. Those who want to work on it, can directly fork this repo, make the necessary changes and then create a pull request**

**Note : This is an open issue, It won't be assigned to anyone.**

**Comment by Shivansh-yadav13** [comment]
@euhidaman can we use bootstrap because some of the UI components are paid in Tailwindcss

**Comment by euhidaman** [comment]
> @euhidaman can we use bootstrap because some of the UI components are paid in Tailwindcss

No, we can't use bootstrap, because the entire front-end part of this website is **already** made using custom Tailwind components.

And, there are many free Tailwind CSS components available too. They are just a google search away !! 😊

**Comment by Shivansh-yadav13** [comment]
Ok 👍
